modules/gapi/src/executor/gstreamingexecutor.cpp

```diff
@@ -680,6 +699,9 @@ class StreamingOutput final: public cv::gimpl::GIslandExecutable::IOutput
     // Prepare this object for posting
     virtual cv::GRunArgP get(int idx) override
     {
+        GAPI_ITT_STATIC_LOCAL_HANDLE(outputs_alloc_hndl, "outputs_allocation_region");
+        GAPI_ITT_AUTO_TRACE_GUARD(outputs_alloc_hndl);
```

Comment:
I am not sure in this trace, it seems extra..

Reply:
This is good to know still if we start allocating 4K images

Reply:
Thanks a lot!